src/components/dialogs/contingency-list-selector.jsx

```diff
@@ -146,30 +141,25 @@ const ContingencyListSelector = (props) => {
     };
 
     const removeFromFavorite = (toRemove) => {
-        const toDelete = new Set(toRemove);
         saveFavorite(
             contingencyList
                 .map((e) => e.id)
-                .filter((item) => !toDelete.has(item))
+                .filter((item) => !toRemove.includes(item))
         );
-        const newChecked = checkedContingencyListUuids.filter(
-            (item) => !toDelete.has(item)
+
+        setCheckedContingencyList((oldCheked) =>
+            oldCheked.filter((item) => !toRemove.includes(item))
```

Comment:
you are checking if an array of ids includes an object (item)

Reply:
Should be Array of Objects as arguments for `addFavorites` and `removeFromFavorites` now